graphql_compiler/macros/__init__.py

```diff
@@ -167,16 +171,25 @@ def get_schema_for_macro_definition(schema):
     """Returns a schema with macro directives.
 
     This returned schema can be used to validate macro definitions, and support GraphQL
-    macro editors, enabling them to autocomplete one the @macro_edge_definition and
-    @macro_edge_target directives.
+    macro editors, enabling them to autocomplete on the @macro_edge_definition and
+    @macro_edge_target directives.*
```

Comment:
Oopsie. Doing `git diff --staged` before you commit will tell you what's going in, and you can spot these kinds of things.

Reply:
That was a typo I fixed. I didn't mention that in my commit message -- I'll do that from now on!